packages/koenig-lexical/src/components/ui/MediaPlaceholderBeta.stories.jsx

```
@@ -0,0 +1,91 @@
+    borderStyle: 'squared'
+};
+
+export const Product = Template.bind({});
+Product.args = {
+    icon: 'product',
+    desc: 'Click to upload a product image',
+    size: 'small',
+    borderStyle: 'squared'
+};
+
+export const Error = Template.bind({});
```

Biome (1.9.4)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)